apps/client/src/services/initialize-socket.ts

```diff
@@ -0,0 +1,11 @@
+import { io, Socket } from 'socket.io-client';
+
+const SOCKET_ENDPOINT =
+  process.env['NX_SOCKET_ENDPOINT'] || 'http://localhost:3000';
```

Comment:
Why do we use NX as the prefix ?

Reply:
Nx checks for the`NX` prefix to add when building.

![image](https://user-images.githubusercontent.com/33794760/221827305-8e7dc495-82a5-448a-82b2-4331dcf6047a.png)
https://nx.dev/recipes/environment-variables/use-environment-variables-in-react